docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3376,6 +3376,25 @@ The same can be achieved by adding this rule:
 
 We recommend to use this kind of exceptions only if it is not possible to change the hiding rule itself. In other cases it is better to change the original rule, using domain restrictions.
 
+:::note CSS injection
+
+In AdGuard products which use **CoreLibs, version 1.18 and later**, element hiding rules may also be used to inject CSS, for example:
```

Comment:
We should also mention version of tsurlfilter that supports this

@scripthunter7

Reply:
```suggestion
In AdGuard products that use **CoreLibs version 1.18 or later**, you can use element hiding rules to inject CSS. For example:
```